mutiny-core/src/node.rs

```diff
@@ -27,7 +27,7 @@ use crate::{lspclient::FeeRequest, storage::MutinyStorage};
 use anyhow::{anyhow, Context};
 use bdk_esplora::esplora_client::AsyncClient;
 use bip39::Mnemonic;
-use bitcoin::hashes::{hex::ToHex, sha256::Hash as Sha256};
+use bitcoin::hashes::{hex::ToHex, sha256, sha256::Hash as Sha256};
```

Comment:
Oh I could make this cleaner

Reply:
was importing `sha256` just to do `sha256::Hash`, replaced that with just using the already imported `Sha256`